### DISCUSSION TITLE
Notifications infinite loading and freezing

### DISCUSSION THREAD

**Discussion by choeqq** [community/community#78282] [Mobile] [answered]
### Select Topic Area

Bug

### How to reproduce ⬇️ ⬅️ 
1. Navigate to Notifications
2. Make long top to bottom swipe gesture
3. Wait a couple of seconds (loader still appears)
4. Navigate to any other tab (loader freezes)

### Description 📝 
- After long vertical swipe to refresh notifications, loader appears, and doesn't seem to go away until reseting the whole app, when navigating to any other tab, loader freezes and doesn't go away 

### Video 🎥 

https://github.com/community/community/assets/69403782/2c85289a-c50f-4f54-83ac-a9526e11c59e

**Comment by stevepopovich** [comment, marked as answer]
Hi @choeqq we have fixes coming out in the next few releases. Thanks for writing in and keep an eye out for future updates!

**Reply by choeqq** [reply]
Sure, thank you for feedback 😉
